app/src/domains/clp/queries/params.ts

```diff
@@ -20,13 +20,15 @@ export const useRewardsParamsQuery = () => {
       enabled: computed(
         () => sifchainClients.queryClientStatus === "fulfilled",
       ),
+      staleTime: minutesToMilliseconds(5),
```

Comment:
@tien - is that right? That'll set it to 5 minutes yeah? We want it set to 5-10 seconds (I'd err on the side of caution and go with 10s).

Reply:
this is specifically for the rewards params rpc only, which should stay the same every reward period? (7 day ish)

Reply:
I can always remove the micro optimization of course since I'm not a fan of doing this either :))

Reply:
Yes.

Where else are we hitting rpc.sifchain? Ideally those queries should be cached too.

Reply:
See here https://github.com/Sifchain/sifchain-ui/pull/614/files#r857958561